components/playlist/renderer/playlist_render_frame_observer.cc

```diff
@@ -35,137 +66,108 @@ void PlaylistRenderFrameObserver::OnDestruct() {
   delete this;
 }
 
-bool PlaylistRenderFrameObserver::EnsureConnectedToMediaHandler() {
-  if (!media_handler_.is_bound()) {
+void PlaylistRenderFrameObserver::AddMediaSourceAPISuppressor(
+    const std::string& media_source_api_suppressor) {
+  DVLOG(2) << __FUNCTION__;
```

Comment:
are all these DVLOGs really necessary? Chromium docs state that most logging should be removed prior to merge

Reply:
They do come in handy sometimes, and are just `DVLOG`s. I also see Chromium doing the same (~10k instances).